README.md

```diff
@@ -3,3 +3,35 @@
 Audio plugin for the RADAR passive remote monitoring app. It uses openSMILE to process audio from
 the phone's microphone, extracts features from it, and sends those features. 
 
+# New Default RadarConfiguration Parameters #
+
+- "audio_duration": 15
```

Comment:
Thanks for updating the readme. 
Could you create a table for the parameters with the unit, default and a description? For an example, see here: https://github.com/RADAR-base/radar-android-phone#sensors